lib/http/response/parser.rb

```diff
@@ -34,17 +34,12 @@ def on_headers_complete(headers)
       end
 
       def on_body(chunk)
-        if @chunk
-          @chunk << chunk
-        else
-          @chunk = chunk
-        end
+        @chunk ||= ''
```

Comment:
this allocates new string.
and honestly i don't see significant improvement here. if/else condition looks easier to read imho in this particular case.